ember/app/components/maintainer-table/template.hbs

```diff
@@ -0,0 +1,28 @@
+<h2 class="table-header">Maintainers</h2>
+<hr class='seperator' />
+{{#if @maintainers}}
+  <table>
+    <thead>
+      <tr>
+        <th>Username </th>
+        <th>Email Address</th>
+      </tr>
+    </thead>
+    <tbody>
+      {{#each (sort-by "isPrimary" @maintainers) as |maintainer|}}
+      <tr>
+        <td>
+          {{maintainer.user.username}}
+        </td>
+        <td>
+          {{maintainer.user.email}}
+        </td>
+      </tr>
+    {{/each}}
+    </tbody>
+  </table>
+{{else}}
+  <div class='none-yet' data-test-versioned-dependencies-none>
+    <span class='text-undefined'>No maintainers yet</span>
```

Comment:
What's the purpose of `text-undefined`?

Reply:
In this case it can be removed (https://github.com/adfinis/Outdated/pull/29/commits/3c76b6cf64b531fbea9b03ff4096d99f4fb1c86f).

Normally it's the result of the [`status-to-class`](https://github.com/adfinis/Outdated/blob/main/ember/app/helpers/status-to-class.js) helper when the status is undefined.